Logging/DebugHandler.php

```diff
@@ -19,30 +19,23 @@
  */
 namespace Buckaroo\Magento2\Logging;
 
-use Monolog\Logger;
+use Magento\Framework\Filesystem\DriverInterface;
 use Magento\Framework\Logger\Handler\Base;
+use Monolog\Logger;
 
 class DebugHandler extends Base
 {
     // @codingStandardsIgnoreLine
     protected $loggerType = Logger::DEBUG;
 
     // @codingStandardsIgnoreLine
-    protected $fileName = '/var/log/Buckaroo/debug.log';
+    protected $fileName = '';
 
-    /**
-     * @inheritDoc
-     */
-    public function write(array $record)
-    {
-        $extra = [
-            'session_id' => '',
-            'customer_id' => '',
-            'quote_id' => '',
-            'order_id' => '',
-        ];
+    public function __construct(
+        DriverInterface $filesystem
+    ) {
+        $this->fileName = '/var/log/Buckaroo/' . date('Y-m-d') . '.log';
```

Comment:
use magento framework `DirectoryList` to retrieve the log dir, instead of hardcoded path.